tests/conftest.py

```diff
@@ -355,8 +396,10 @@ async def two_nodes_sim_and_wallets():
 
 
 @pytest_asyncio.fixture(scope="function")
-async def two_nodes_sim_and_wallets_services():
-    async for _ in setup_simulators_and_wallets_service(2, 0, {}):
+async def two_nodes_sim_and_wallets_services(blockchain_constants):
+    async for _ in setup_simulators_and_wallets_service(
+        2, 0, {"SOFT_FORK3_HEIGHT": blockchain_constants.SOFT_FORK3_HEIGHT}
```

Comment:
same comment here. doing it like this will run identical tests multiple times. e.g. `Mode.PLAIN` and `Mode.HARD_FORK2_0` will cause the exact same setup, and run any tests that depend on this twice with identical constants.